python/lsst/ts/m2gui/control_tabs.py

```diff
@@ -90,7 +89,6 @@ def _setup_tabs_and_list_widget(self, model):
             TabDetailedForce("Detailed Force", model),
             TabDiagnostics("Diagnostics", model),
             TabAlarmWarn("Alarms/Warnings", model),
-            TabLimitSwitchStatus("Limit Switch Status", model),
```

Comment:
Is there any need to pass the title as a parameter to the constructor? Why this cannot be done in a class itself?

Reply:
Also a general comment (after running the GUI): you need essentially two clicks to show what can be done with a single click. I understand you would like to have UI layout similar to LabVIEW, but for user ergonomics, it's much better to drop "Show selected table" button from control_tabs and make options in listing buttons, which when clicked will show the table.

Reply:
For the question one, I felt it is easier to pass the title a parameter that I do not need to call the `setWindowTitle()` for each class to avoid the duplication of code.

For the question two, I tried this design before and I did not like the idea that when I click the item, the widget will show by itself. Therefore, I ended to decide to have a single button for this.

Thanks!

Reply:
You don't have to call setWindowTitle. Those windows have their names fixed, you can pass those in tab_alarm_warn at constructor, when you call super().__init__(..